src/components/user-login/user-login.module.scss

```diff
@@ -21,12 +21,15 @@
 
 body .dropdown {
```

Comment:
why `body` is here?

Reply:
without it for some reasons my class does not override default one (even though my class comes second in a className list). and I can not define any other child except `body`, because drop down is opened in portal

Reply:
just checked again and now it works as expected... not sure why this happened, removing `body`